src/Passes/Source/ValidationPass/ValidationConfiguration.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ConfigurationManager.hpp"
+#include "Types/Types.hpp"
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    class ValidationPassConfiguration
```

Comment:
Consider renaming file to 
Validation**Pass**Configuration.hpp

Reply:
Renamed.